R/get_source_expressions.R

```diff
@@ -309,12 +308,13 @@ next_with_parent <- function(pc, loc) {
 
   loc <- which(with_parent$id == id)
 
-  return(which(pc$id == with_parent$id[loc + 1L]))
+  which(pc$id == with_parent$id[loc + 1L])
 }
 
 top_level_expressions <- function(pc) {
   if (is.null(pc)) {
-    return(NULL)
+    character()
```

Comment:
This function is returning the output of `which()` from the other branch, which is a integer vector, so this should be `integer()`. You are correct the previous return value of `NULL` was incorrect.